front/app/containers/Admin/dashboard/visitors/index.tsx

```diff
@@ -29,6 +31,17 @@ const query: Query = {
   },
 };
 
+const getDefaultStartDate = (firstSessionDate: string | null) => {
+  if (!firstSessionDate) return moment(LOWER_BOUND);
+
+  const defaultStartDate =
```

Comment:
The default start date for the visitors dashboard will be the date we started collecting pageview / more rich session data, back in November 2024, EXCEPT if the platform was created after that (i.e. if the first recorded session if after that)